executor/analyze.go

```diff
@@ -725,9 +725,6 @@ func (e *AnalyzeColumnsExec) buildSamplingStats(ranges []*ranger.Range) (
 			if err != nil {
 				return 0, nil, nil, nil, err
 			}
-			if sample.Columns[i].Kind() == types.KindBytes {
-				sample.Columns[i].SetBytes(sample.Columns[i].GetBytes())
-			}
```

Comment:
If I understood it correctly, this step is needed for column stats and not needed for index stats. :exploding_head: